src/components/dashboard.jsx

```diff
@@ -49,9 +49,9 @@ const ListGroupWithMore = React.createClass({
       </BS.ListGroup>
     );
   }
-});
+}
 
-const RepoItem = React.createClass({
+class RepoItem extends React.Component {
   render() {
```

Comment:
This can be converted to functional component.

All component with only render method can be functional component, I'll stop marking.